### PR TITLE
Declared volumes to projects, logs and apache2 configuration

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hello,

I'd like to be able to bind volumes using Kitematic in Windows or Mac OSX, to use your image for learning purposes.

What do you think ?

Mickaël

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ MAINTAINER Ronan Guilloux <ronan.guilloux@gmail.com>
 # Add some utilities
 ADD docker/pim-download.sh /usr/local/bin/pim-download
 ADD docker/pim-initialize.sh /usr/local/bin/pim-initialize
+ADD dpcker/pim-declare-volumes.sh /usr/local/bin/pim-declare-volumes
```
Comment (ronanguilloux) [inline Dockerfile]: s/dpcker/docker ;-)

**Review by ronanguilloux — COMMENTED** [review]
small typo

**Comment by mickaelandrieu** [comment]
shame on me! fixed

**Comment by mickaelandrieu** [comment]
To be honest, I wanted to declare volumes in DockerFile file directly : sounds overkill to add a new file for 1 instruction ^^

If you like the idea I can update and squash my commits

**Comment by mickaelandrieu** [comment]
@ronanguilloux friendly reminder: declare volumes entries allow anyone to edit project code with any IDE, this could make this docker image great for learning/teaching purposes :)

File: docker/pim-declare-volumes.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+VOLUME ["/home/docker/pim", "/var/log/apache2", "/etc/apache2"]
```
Comment (ronanguilloux) [inline docker/pim-declare-volumes.sh:3]: There's a "no newline at end of file" msg in Github here, I suppose it's a must have

Comment (ronanguilloux) [reply docker/pim-declare-volumes.sh:3]: I think you should give an example in the README about how to take advantage of the VOLUME you set up

Comment (mickaelandrieu) [reply docker/pim-declare-volumes.sh:3]: yes, I will complete my pull request

**Pull request closed without merging**